src/host/_stream.cpp

```diff
@@ -295,25 +421,16 @@ try
         return CONSOLE_STATUS_WAIT;
     }
 
-    const auto vtIo = ServiceLocator::LocateGlobals().getConsoleInformation().GetVtIo();
     const auto restoreVtQuirk = wil::scope_exit([&]() {
         if (requiresVtQuirk)
         {
             screenInfo.ResetIgnoreLegacyEquivalentVTAttributes();
         }
-        if (vtIo->IsUsingVt())
-        {
-            vtIo->CorkRenderer(false);
-        }
     });
     if (requiresVtQuirk)
     {
         screenInfo.SetIgnoreLegacyEquivalentVTAttributes();
     }

```

Comment:
This may be something we can leave for a followup PR, but as far as I can tell, this quirk doesn't actually work anymore, and can possibly be removed (and if we do that, there's actually quite a lot of associated code that could get dumped).

My understanding was that the main reason for this was to map SGR 37/40 sequences (technically the current default indices) into SGR 39/49. This was handled in the `SCREEN_INFORMATION::SetAttributes` which was called from the conhost implementation of the `ITerminalApi::SetTextAttributes` method (in fact that was was the only reason we even need to have that method in the api).

But with the VT sequences now being passed through directly, that mapping isn't happening anymore, and the result is that Windows PowerShell is getting some weird color glitches again when you have a non-standard background color. For example, this is with Campbell Powershell color scheme:

![image](https://github.com/user-attachments/assets/0e03d9de-ca6f-405d-a13a-d8a36c849e23)

One thing we could try as a simple fix is to send through a `DECAC` sequence on startup if that quick flag is set. You can see the effect this has in PowerShell by doing it manually with `echo "$([char]27)[1;7;0,|"`. You'll lose the blue background, because the default colors are now white on black, but at least that would avoid the weird mix of blue and black. I'm not sure it'll covers all cases though.

Anyway, I just wanted to raise the issue so you're aware of the problem. Doesn't necessarily need to be resolved now.

Reply:
This issue was fixed in PSReadLine at around the same time that `SetIgnoreLegacyEquivalentVTAttributes` got (re-)added. It's been ~4 years since then and not just Windows 11 but also 10 now ship PSReadLine 2.0.0 which contains the bug fix for this. I think we could honestly just remove it now.

Reply:
I'd be thrilled for us to remove that code, even without the bugfix, but I just want to note that I am definitely still seeing that issue in Windows PowerShell (not PowerShell 7 though).

My Windows version is 10.0.19045.4651 (which is the latest available to me), and `Get-Module PSReadline` indicates the PSReadline version is 2.0.0, but I'm not a regular PowerShell user, so it's possible I'm just holding it wrong.

Reply:
I saw that it used "2.0.0", just like my Windows 11, so I just assumed it's the same. I didn't realize that there's a hidden suffix...

![image](https://github.com/user-attachments/assets/23f35cb7-d5ba-48b8-b56f-6baf7f1bd3a3)

I think we're indeed hosed.